src/test/java/com/coinbase/exchange/api/orders/OrderTests.java

```diff
@@ -102,10 +102,29 @@ public void getAllOpenOrders() {
     }
 
     @Test
-    public void getFills() {
-        List<Fill> fills = orderService.getAllFills();
+    public void getFillsByProductId() {
+        List<Fill> fills = orderService.getFillsByProductId("BTC-USD", 100);
         assertTrue(fills.size() >= 0);
     }
+
+    @Test
+    public void createMarketOrderBuyThenGetFillByOrderId() {
+        NewMarketOrderSingle marketOrder = createNewMarketOrder("BTC-USD", "buy", new BigDecimal(0.01));
+        Order order = orderService.createOrder(marketOrder);
+
+        assertTrue(order != null); //make sure we created an order
+        String orderId = order.getId();
+        assertTrue(orderId.length() > 0); //ensure we have an actual orderId
```

Comment:
perhaps assertNotNull works better here - so long as we get back something then we don't really care about the value?

Reply:
Agree, will change